src/verinfast/config.py

```diff
@@ -389,11 +387,10 @@ def handle_args(self, args):
             self.runGit = args.should_git
 
         if "truncate_findings" in args and args.truncate_findings is not None:
-            if args.truncate_findings > 0:
-                self.truncate_findings = True
```

Comment:
Should leave this. I don't want to change VF behavior just always pass truncate_findings: true and truncate_findings_length: 0 in the config

Reply:
Fixed @aylusltd . Other annotations in the code here.